readthedocs/search/views.py

```diff
@@ -80,7 +80,6 @@ def elastic_search(request, project_slug=None):
     Global user search on the dashboard.
 
     This is for both the main search and project search.
-
```

Comment:
Riga presente nel file originale di RTD, ripristinarlo per facilitare in futuro l'allineamento con quel repository

Reply:
errore mio, ripristinata